packages/cli/src/cli/ink-cli/utils/input.ts

```
@@ -0,0 +1,59 @@
+
+export const ESC = '\u001B';
+export const SGR_EVENT_PREFIX = `${ESC}[<`;
+export const X11_EVENT_PREFIX = `${ESC}[M`;
+
+// Focus tracking sequences
+export const FOCUS_IN = `${ESC}[I`;
+export const FOCUS_OUT = `${ESC}[O`;
+
+// Mouse event regex patterns
+// eslint-disable-next-line no-control-regex
+export const SGR_MOUSE_REGEX = /^\x1b\[<(\d+);(\d+);(\d+)([mM])/;
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)